src/condition.cpp

```diff
@@ -915,7 +915,7 @@ bool ConditionRegeneration::executeCondition(Creature* creature, int32_t interva
 					message.type = MESSAGE_HEALED_OTHERS;
 					message.text = player->getName() + " was healed for " + healString;
 					for (Creature* spectator : spectators) {
-						spectator->getPlayer()->sendTextMessage(message);
+						static_cast<Player*>(spectator)->sendTextMessage(message);
```

Comment:
Please add `assert(dynamic_cast<Player*>(spectator) != nullptr)` before so we catch bugs on debug builds